mod_reforged/hooks/entity/tactical/humans/gunner.nut

```diff
@@ -20,31 +20,7 @@
 		// this.m.Skills.add(this.new("scripts/skills/actives/recover_skill"));	// Now granted to all humans by default
 
 		// Reforged
-		b.RangedDefense += 10;
 		this.m.Skills.add(::new("scripts/skills/perks/perk_mastery_crossbow"));
 		this.m.Skills.add(::new("scripts/skills/perks/perk_rotation"));
-		this.m.Skills.add(::new("scripts/skills/perks/perk_rf_entrenched"));
-		this.m.Skills.add(::new("scripts/skills/perks/perk_rf_power_shot"));
-		this.m.Skills.add(::new("scripts/skills/perks/perk_rf_through_the_ranks"));
```

Comment:
Same logic as the mercenary_ranged, I think Through the Ranks should be maintained on these guys (especially for allied gunners with players in the field).

Reply:
resolved by giving through the ranks back